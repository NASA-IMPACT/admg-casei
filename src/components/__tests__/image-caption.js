import React from "react"
import renderer from "react-test-renderer"
import ImageCaption from "../image-caption"

describe("ImageCaption", () => {
  it("renders caption on image", () => {
    const tree = renderer
      .create(
        <div style={{ position: `relative` }}>
          <img
            src="https://picsum.photos/300/300"
            alt="test-image"
            data-cy="test-image"
          />
          <ImageCaption id="test-image">caption</ImageCaption>
        </div>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
