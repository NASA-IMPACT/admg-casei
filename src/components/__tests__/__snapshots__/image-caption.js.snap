// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageCaption renders caption on image 1`] = `
<div
  style={
    Object {
      "position": "relative",
    }
  }
>
  <img
    alt="test-image"
    data-cy="test-image"
    src="https://picsum.photos/300/300"
  />
  <label
    data-cy="test-image-caption"
    style={
      Object {
        "left": "1.5rem",
        "maxWidth": "24rem",
        "position": "absolute",
        "textShadow": "1px 1px rgba(0,0,0,0.8)",
        "top": "16rem",
      }
    }
  >
    caption
  </label>
</div>
`;
